core/metacling/src/TCling.cxx

```diff
@@ -2678,6 +2678,14 @@ void TCling::InspectMembers(TMemberInspector& insp, const void* obj,
       return;
    }
 
+   if (TClassEdit::IsUniquePtr(cl->GetName())) {
+      // Ignore error caused by the inside of std::unique_ptr
+      // This is needed solely because of rootclingIO's IsUnsupportedUniquePointer
+      // which check the numbers of data members.
+      // If this usage is remove, this can be replace with a return statement.
+      isTransient = true;
```

Comment:
Could you point out where it is further down in this function that the `isTransient == true` prevents an error?

Reply:
It is the 'whole' thing; this function is used (indirectly) to populate the "list of real data". If rootclingIO's IsUnsupportedUniquePointer was not looking at that list, we could afford to keep it empty.